pyqtorch/utils.py

```diff
@@ -752,8 +750,13 @@ def is_parametric(operation: pyq.Sequence) -> bool:
     for m in operation.modules():
         if isinstance(m, (pyq.Scale, Parametric)):
             params.append(m.param_name)
-
     res = False
-    if any(isinstance(p, str) for p in params):
+    if any(isinstance(p, (str, pyq.ConcretizedCallable)) for p in params):
         res = True
     return res
+
+
+def heaviside(x: Tensor, _: Tensor) -> Tensor:
+    a = torch.zeros(2)
+    a[0] = x
+    return torch.clamp(1000 * torch.max(a), torch.tensor(0.0), torch.tensor(1.0))
```

Comment:
Can you add a docstring explaining why this is needed, and add some tests that cover the intended use?

Reply:
Removed the `ConcretizedCallable` stuff from `parametric.py`. It's not actually needed